api/opds.py

```diff
@@ -556,3 +569,40 @@ def single_hold_feed(cls, circulation, hold, test_mode=False):
                         active_holds_by_work={work:hold}, 
                         test_mode=test_mode)
         return AcquisitionFeed.single_entry(db, work, annotator)
+
+
+class PreloadFeed(AcquisitionFeed):
+
+    @classmethod
+    def page(cls, _db, title, url, annotator=None,
+             use_materialized_works=True):
+
+        """Create a feed of content to preload on devices."""
+        configured_content = Configuration.policy(Configuration.PRELOADED_CONTENT)
+
+        identifiers = [Identifier.parse_urn(_db, urn)[0] for urn in configured_content]
+        identifier_ids = [identifier.id for identifier in identifiers]
+
+        if use_materialized_works:
```

Comment:
This seems like copy-and-pasted code from Lane.materialized_works. Is it possible to refactor Lane.materialized_works to avoid the duplication?

Not a big deal and not necessary now, especially if this is just code you copied from content to circulation.